toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/panels/FileChoosePanel.java

```diff
@@ -0,0 +1,113 @@
+package io.toast.tk.agent.ui.panels;
+
+import io.toast.tk.agent.config.AgentConfigProvider;
+import io.toast.tk.agent.ui.i18n.UIMessages;
+import io.toast.tk.agent.ui.utils.ConfigTesterHelper;
+import io.toast.tk.agent.ui.utils.PanelHelper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.swing.*;
+import javax.swing.border.TitledBorder;
+import java.awt.*;
+import java.awt.event.ActionEvent;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.awt.event.ActionEvent'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)